runner/src/server/plugins/engine/plugin.ts

```diff
@@ -220,7 +240,12 @@ export const plugin = {
       method: "get",
       path: "/{id}/{path*}",
       options: {
-        auth: jwtAuthStrategyIsActive ? jwtAuthStrategyName : options.auth,
+        auth: jwtAuthStrategyIsActive
+          ? {
+              mode: "required",
+              strategies: ["simple", jwtAuthStrategyName],
```

Comment:
just to firm up my understanding - are we using "simple" only here as its the first in the list?

Reply:
thanks Harry 👌